stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
-appVersion: 3.8.5
-version: 1.46.3
+appVersion: 3.8.6
```

Comment:
Please don't bump the appVersion as it's still at rabbitmq 3.8.5

Reply:
Sorry about that, this is my first time doing this, I updated the commit